spec/features/orders/visitor_can_checkout_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe 'Visitor can checkout cart' do
+  before(:each) do
+    @test_user1 = create(:user)
+    @outing1, @outing2, @outing3, @outing4 = create_list(:outing, 4)
+  end
+  scenario 'after logging in' do
+    visit outings_path
+
+    within '#outing-index:nth-of-type(1)' do
+      click_on "Add to Cart"
+    end
+    within '#outing-index:nth-of-type(2)' do
+      click_on "Add to Cart"
+    end
+
+    click_on 'View Cart'
+
+    expect(current_path).to eq(cart_path)
+    expect(page).to have_content(@outing1.title)
+    expect(page).to have_content(@outing2.title)
+
+    click_on 'Checkout'
```

Comment:
@drod1000 @Laszlo-JFLMTCO @njgheorghita if lazlo implemented the logout feature, with i think he did, then #23 should be all done.